test/SlowTests/Smuggler/BackupDatabaseRecordTests.cs

```diff
@@ -39,6 +40,13 @@ public BackupDatabaseRecordTests(ITestOutputHelper output) : base(output)
         public async Task CanExportAndImportDatabaseRecord()
         {
             var file = Path.GetTempFileName();
+            var dummy = GenerateAndSaveSelfSignedCertificate(true);
+            string privateKey;
+            using (var pullReplicationCertificate =
+                new X509Certificate2(dummy.ServerCertificatePath, (string)null, X509KeyStorageFlags.MachineKeySet | X509KeyStorageFlags.Exportable))
```

Comment:
Is this correct? To dispose the certificate created by `GenerateAndSaveSelfSignedCertificate` ? It is used by the whole test suite. Maybe it is fine, because we are creating a new instance here, but I would like to double check.

Reply:
I think it is fine, since we use `dummy.ServerCertificatePath` (which is a path and not the certificate itself).